sources/packages/api/src/types/config.ts

```diff
@@ -1,10 +1,20 @@
 export interface IConfig {
   auth: IAuthConfig;
+  e2eTest: Ie2eTest;
```

Comment:
Can we move this into test or testHelpers?

Reply:
Done!